addon/components/goal/goal-index.js

```diff
@@ -1,12 +1,31 @@
 import layout from 'bepstore-goals/templates/components/goal/goal-index';
 import Ember from 'ember';
 
-export default Ember.Component.extend({
+
+const { Component, inject: { service } } = Ember;
+
+export default Component.extend({
 	layout,
+  session: service(),
+
+	addType:null,
+
+	isCoreMember: Ember.computed('session.user', function(){
+		let id = this.get('session.user.id');
+		return (id === this.get('model.product_owner.id')) || (id === this.get('model.head_design.id')) || (id === this.get('model.head_frontend.id')) || (id === this.get('model.head_backend.id'));
```

Comment:
I don't think so
OR stops when 1 of the statements is true, so if you are the product owner, it stops after the fist statement/get.
In your code, all will be loaded every time